content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -1,153 +0,0 @@
-# DVCLive with DVC
-
-Even though DVCLive does not require DVC, they can integrate in a couple useful
-ways:
-
-- The [outputs](#outputs) DVCLive produces are recognized by `dvc exp`,
-  `dvc metrics` and `dvc plots`. Those same outputs can be visualized in
-  [Iterative Studio](#iterative-studio).
-
-- DVCLive is also capable of generating [checkpoint](#checkpoints) signal files
-  used by DVC <abbr>experiments<abbr>.
```

Comment:
This summary was nice. If we remove it, should we mention these things more in DVC docs and link to the new DVCLive how-tos?

Reply:
I believe the first bullet point is now covered in the get started https://dvc-org-dvclive-refacto-czwjzz.herokuapp.com/doc/dvclive/get-started#dvc

Regarding the second bullet point, there is a detailed page about checkpoints in DVC user guide and it already mentions/points to DVCLive.

Reply:
Agree with @jorgeorpinel that I liked having a summary of all the DVC magic that DVCLive adds. Otherwise, it feels like readers aren't quite sure what DVCLive is doing.